learning_journal/learning_journal/models.py

```diff
@@ -0,0 +1,28 @@
+import datetime
+from sqlalchemy import (
+    Column,
+    Index,
+    Integer,
+    Unicode,
+    UnicodeText,
+    DateTime,
+)
+from sqlalchemy.ext.declarative import declarative_base
+from sqlalchemy.orm import (
+    scoped_session,
+    sessionmaker,
+)
+from zope.sqlalchemy import ZopeTransactionExtension
+
+DBSession = scoped_session(sessionmaker(extension=ZopeTransactionExtension()))
+Base = declarative_base()
+
+
+class Entry(Base):
+    __tablename__ = 'entries'
+    id = Column(Integer, unique=True, primary_key=True)
```

Comment:
primary key fields are always unique, the use of the unique constraint here is redundant.